README.md

```diff
@@ -1,2 +1,13 @@
 # arxiv_oaipmh
 handles requests to the OAI PMH service
+
+# run locally
+install dependencies:
+poetry install
+
+run:
+python main.py
+
```

Comment:
Usually it is nice to have something like

"Then check the app is running with http://localhost:8080/oai/something"